HW4/task1.py

```diff
@@ -0,0 +1,13 @@
+text = input("write a text:")
+
+if not text.isalpha():
+    print("this is a number")
+
+    if (int(text) % 2) == 0:
```

Comment:
nit: bracks can be omitted
```suggestion
    if int(text) % 2 == 0:
```